backupwordpress.php

```diff
@@ -76,7 +76,7 @@ public static function get_instance() {
 	 */
 	public function plugins_loaded() {
 
-		if ( false !== $this->maybe_self_deactivate() ) {
```

Comment:
Yes - maybe the naming is confusing?
If we need to self deactivate it returns true.